[2024-05-02T07:54:02.828+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-02T07:54:02.848+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3-data-transfer-dag.transfer-order manual__2024-05-02T07:53:57.048479+00:00 [queued]>
[2024-05-02T07:54:02.854+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3-data-transfer-dag.transfer-order manual__2024-05-02T07:53:57.048479+00:00 [queued]>
[2024-05-02T07:54:02.855+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-02T07:54:02.865+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transfer-order> on 2024-05-02 07:53:57.048479+00:00
[2024-05-02T07:54:02.874+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=256) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-02T07:54:02.875+0000] {standard_task_runner.py:63} INFO - Started process 267 to run task
[2024-05-02T07:54:02.876+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3-data-transfer-dag', 'transfer-order', 'manual__2024-05-02T07:53:57.048479+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/new-dag.py', '--cfg-path', '/tmp/tmpenj61_1m']
[2024-05-02T07:54:02.877+0000] {standard_task_runner.py:91} INFO - Job 62: Subtask transfer-order
[2024-05-02T07:54:02.914+0000] {task_command.py:426} INFO - Running <TaskInstance: s3-data-transfer-dag.transfer-order manual__2024-05-02T07:53:57.048479+00:00 [running]> on host 0a71bd2d4f31
[2024-05-02T07:54:02.981+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3-data-transfer-dag' AIRFLOW_CTX_TASK_ID='transfer-order' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T07:53:57.048479+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-02T07:53:57.048479+00:00'
[2024-05-02T07:54:02.982+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-02T07:54:04.433+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-02T07:54:04.433+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-02T07:54:04.441+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=s3-data-transfer-dag, task_id=transfer-order, execution_date=20240502T075357, start_date=20240502T075402, end_date=20240502T075404
[2024-05-02T07:54:04.493+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-02T07:54:04.504+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-02T07:54:04.506+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
